k8-util/helm/fluvio-sys/templates/crd_mirror.yaml

```diff
@@ -46,14 +46,17 @@ spec:
                           type: string
                         publicEndpoint:
                           type: string
-                keyPair:
-                  type: object
-                  required: ["privateKey", "publicKey"]
-                  properties:
-                    publicKey:
-                      type: string
-                    privateKey:
-                      type: string
+                        tls:
```

Comment:
Should this be a secret?

Reply:
Is this metadata used by the edge or home? Why is `tls` config needed here?

Reply:
> Is this metadata used by the edge or home? 

Edge is storing it as Home. 


> Why is tls config needed here?

To connect to home with TLS.